Logger/BaseLoggerMixins.cs

```diff
@@ -1,6 +1,56 @@
-﻿namespace Logger;
+﻿namespace Logger;
+using System.Globalization;
 
 public static class BaseLoggerMixins
 {
+    public static void Error_WithNullLogger_ThrowsException(this BaseLogger? logger, string message, params object[] args)
```

Comment:
This can be just called `Error`. Same applies for other methods.

```suggestion
    public static void Error(this BaseLogger? logger, string message, params object[] args)
```